### PR TITLE
fix(images): update digest field for a mapped image

### PR DESCRIPTION
**Pull request by vdice**
# What does this change
* Updates the (former) `digest` field for a mapped image to `contentDigest`, per the CNAB core spec
* Also updates the similar field for an invocation image in some test data files

# What issue does it fix
Closes https://github.com/getporter/porter/issues/1307

# Notes for the reviewer

# Checklist
- [x] Unit Tests
- [ ] Documentation
- [x] Schema (porter.yaml)

If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

[contributors]: /CONTRIBUTORS.md

### CODE REVIEW

File: pkg/runtime/runtime-manifest.go [hunk 1]
```diff
@@ -376,7 +376,7 @@ func (m *RuntimeManifest) buildSourceData() (map[string]interface{}, error) {
 		typeOfT := val.Type()
 		for i := 0; i < val.NumField(); i++ {
 			f := val.Field(i)
-			name := strings.ToLower(typeOfT.Field(i).Name)
+			name := toLowerCase(typeOfT.Field(i).Name)
```
Comment (carolynvs) [inline pkg/runtime/runtime-manifest.go:379]: Oh so this "jsonify's" the name. Previously we were making the entire name lower case which would mess up something like "MyThing" turning it into "mything" when it should be "myThing". Do I have that right?

File: pkg/runtime/runtime-manifest.go [hunk 2]
```diff
@@ -518,3 +518,13 @@ func resolveImage(image *manifest.MappedImage, refString string) error {
 	}
 	return nil
 }
+
+// toLowerCase returns a string with the first character lowercased
+func toLowerCase(str string) string {
```
Comment (carolynvs) [inline pkg/runtime/runtime-manifest.go:523]: The name of this function is misleading. Perhaps it should be called `toCamelCase`?

https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

**Review by carolynvs — CHANGES_REQUESTED** [review]
I'm glad you noticed that digest wasn't conformant anymore. Thanks for fixing this!

I just had one question about how the struct name was being turned into a string, then this is good to go.

File: pkg/runtime/runtime-manifest.go (hunk 1 above)
Comment (vdice) [reply pkg/runtime/runtime-manifest.go:379]: Yup, that's correct!

File: pkg/runtime/runtime-manifest.go (hunk 2 above)
Comment (vdice) [reply pkg/runtime/runtime-manifest.go:523]: 👍 `toCamelCase` it shall be.

**Review by carolynvs — APPROVED** [review]
LGTM 👍

**Pull request merged**